src/deluge/gui/views/session_view.cpp

```diff
@@ -4073,6 +4074,7 @@ ActionResult SessionView::gridHandlePadsLaunch(int32_t x, int32_t y, int32_t on,
 				gridFirstPressedY = y;
 				// will create the track if it doesn't exist
 				clip = gridCreateClip(gridSectionFromY(y), track, nullptr);
+				transitionToViewForClip(clip);
```

Comment:
I don't think this is desirable in launch mode, it gets in the way of live looping